pydantic/utils.py

```diff
@@ -113,6 +113,13 @@
     deque,
 }
 
+# There are error classes caught in smart_deepcopy
+CAUGHT_SMART_DEEPCOPY_ERRORS: Set[Type[Exception]] = {
```

Comment:
I don't think you need to define this separately, just hard code it in the except block.

Reply:
then manually parameterise the test to the same exception types.

Reply:
https://github.com/pydantic/pydantic/pull/4187/commits/624a0aa0be4a15923907501d90e516e6e5b778d4